specification/azurestack/resource-manager/Microsoft.AzureStack/preview/2020-06-01-preview/EdgeSubscription.json

```diff
@@ -0,0 +1,398 @@
+              "$ref": "examples/EdgeSubscription/GetEdgeSub.json"
+            }
+          },
+          "tags": [
+            "EdgeSubscriptions"
+          ],
+          "description": "Returns the properties of an Azure Stack edge subscriptions.",
+          "operationId": "EdgeSubscriptions_Get",
+          "parameters": [
+            {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+            },
+            {
+              "$ref": "#/parameters/ResourceGroupNameParameter"
+            },
+            {
+                "$ref": "#/parameters/EdgeSubscriptionNameParameter"
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK",
+              "schema": {
+                "$ref": "#/definitions/EdgeSubscription"
+              }
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            }
+          },
+          "x-ms-pageable": {
+            "nextLinkName": "nextLink"
+          }
+        },
+        "delete": {
+          "x-ms-examples": {
+            "Delete the requested Azure Stack edge subscription.": {
+              "$ref": "examples/EdgeSubscription/DeleteEdgeSub.json"
+            }
+          },
+          "tags": [
+            "EdgeSubscriptions"
+          ],
+          "description": "Delete the requested Azure Stack edge subscription.",
+          "operationId": "EdgeSubscriptions_Delete",
+          "parameters": [
+            {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+            },
+            {
+              "$ref": "#/parameters/ResourceGroupNameParameter"
+            },
+            {
+              "$ref": "#/parameters/EdgeSubscriptionNameParameter"
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK"
+            },
+            "204": {
+              "description": "NO CONTENT"
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            }
+          }
+        },
+        "put": {
+          "x-ms-examples": {
+            "Create or update an Azure Stack registration.": {
+              "$ref": "examples/EdgeSubscription/PutEdgeSub.json"
+            }
+          },
+          "tags": [
+            "EdgeSubscriptions"
+          ],
+          "description": "Create a new edge subscription",
+          "operationId": "EdgeSubscriptions_Create",
+          "parameters": [
```

Comment:
>parameters [](start = 11, length = 10)

There doesn't seem to be a request body parameter.

Reply:
Added the request body param NewEdgeSubscriptionParameter